helix-query.yaml

```diff
@@ -33,6 +33,13 @@ indices:
       category:
         select: head > meta[name="category"]
         value: attribute(el, "content")
+      lastModified:
```

Comment:
@mayur-satav Change this to 
`      lastModified:
        select: none
        value: parseTimestamp(headers["last-modified"], "ddd, DD MMM YYYY hh:mm:ss GMT")`